client/py_client/client.py

```diff
@@ -217,3 +236,9 @@ def get_bootstrappers_with_remaining_newbie_tickets(self, cid):
         ret = subprocess.run(self.cli + ["get-bootstrappers-with-remaining-newbie-tickets", "--cid", cid],
                              stdout=subprocess.PIPE)
         return ret.stdout.decode("utf-8").strip()
+
+    def transfer_all(self, cid, source, dest, pay_fees_in_cc=False):
```

Comment:
Looks very good, but can you add the `pay_fees_in_cc` for all commands that send an extrinsic here. We will use that in the future for better integration tests.

If I saw correctly, you already check this CLI on the rust side already anyhow. :)

Reply:
i ended up refactoring the whole thing, see 44ad2d9845e89fc0d2cf29acf2db2b0a8fb5c807